src/main/osd/osd_elements.c

```diff
@@ -1127,6 +1127,55 @@ static void osdElementEfficiency(osdElementParms_t *element)
         tfp_sprintf(element->buff, "----%c/%c", SYM_MAH, unitSymbol);
     }
 }
+
+int16_t osdGetBoundaryFromDisplayX(int16_t error_x)
```

Comment:
```suggestion
int16_t osdGetBoundaryFromDisplay(uint8_t fov, int16_t error)
```
You are repeating the same code twice. Instead, you can use one generalized function (for example with the parameters I suggested)